apps/sms/index.html

```diff
@@ -113,6 +113,7 @@
     <script defer src="js/contact_renderer.js"></script>
     <script defer src="js/information.js"></script>
     -->
+    <link rel="import" href="waiting-screen.html">
```

Comment:
unfortunately it seems HTML Imports will not ship :( https://hacks.mozilla.org/2014/12/mozilla-and-web-components/